en/techdocs/api/client/hoodie.md

```diff
@@ -283,22 +369,37 @@ chat.findAll('message')
 
 **version:**      *> 0.2.0*
 
-*Extend the hoodie API with new functionality*
-
-<pre><code>hoodie.extend(plugin);</code></pre>
+**Extend the hoodie API with new functionality**
 
-| option     | type     | description     | required |
-| ---------- |:--------:|:---------------:|:--------:|
-| plugin     | function | extend the frontend API with whatever logic your plugin wants to expose | yes |
+```javascript
+hoodie.extend(plugin);
+```
 
+| option     | type     | description                                                             | required |
+|:---------- |:-------- |:----------------------------------------------------------------------- |:-------- |
+| plugin     | function | extend the frontend API with whatever logic your plugin wants to expose | yes      |
 
+Used when writing Hoodie plugins, gives you the means to extend the frontend API of Hoodie to expose methods to use your plugin. [Check out this detailed guide to writing your own plugins](/en/plugins/tutorial.html).
 
 ##### Example
-<pre><code>hoodie.extend(function(hoodie, lib, util) {
+```javascript
+hoodie.extend(function(hoodie, lib, util) {
   hoodie.sayHi = function() {
     alert('say hi!');
   };
 });
 
 hoodie.sayHi(); // shows alert
-</code></pre>
+
+```
+
+
+## Onward!
+
+Now head on over to the exciting stuff: the [account](/en/techdocs/api/client/hoodie.account.html) and [store](/en/techdocs/api/client/hoodie.store.html) docs! Learn how to sign up users and let them store data, any many other cool things.
+
+We hope this API guide was helpful! If not, please let us help you <a href="http://hood.ie/chat" target="_blank">on IRC or Slack</a>.
+
+We also have an <a href="http://faq.hood.ie" target="_blank">FAQ</a> that could prove useful if things go wrong.
+
+If you find this guide in error or out of date, you could also <a href="https://github.com/hoodiehq/documentation/issues" target="_blank">open an issue</a> or <a href="https://github.com/hoodiehq/documentation/pulls" target="_blank">submit a pull request</a> with your corrections to [this file](https://github.com/hoodiehq/documentation/blob/gh-pages/en/techdocs/api/client/hoodie.md).
```

Comment:
I'm not sure if linking to `https://github.com/hoodiehq/documentation/pulls` is helpful here. I think I'd leave it out